sa/storage-authority.go

```diff
@@ -897,18 +897,13 @@ func (ssa *SQLStorageAuthority) GetSCTReceipt(serial string, logID string) (rece
 	return
 }
 
-// ErrDuplicateReceipt is an error type for duplicate SCT receipts
-type ErrDuplicateReceipt string
-
-func (e ErrDuplicateReceipt) Error() string {
-	return string(e)
-}
-
 // AddSCTReceipt adds a new SCT receipt to the (append-only) sctReceipts table
 func (ssa *SQLStorageAuthority) AddSCTReceipt(sct core.SignedCertificateTimestamp) error {
 	err := ssa.dbMap.Insert(&sct)
+	// For AddSCTReceipt, duplicates are explicitly OK, so don't return errors
+	// based on duplicates.
 	if err != nil && strings.HasPrefix(err.Error(), "Error 1062: Duplicate entry") {
```

Comment:
You might want to add to this comment to say that currently when we try to re-submit we will get one of these per log we've already submit to, so it's spammy. Once we finally implement the 'only submit to the missing logs' patch (blocked on #891) having something like this would be quite useful as we'll want to know when we are double-submitting for no real reason.